### ISSUE TITLE
modular80

### ISSUE THREAD

**Issue by cschol** [VCVRack/library#483]
My plugin has been updated:
https://github.com/cschol/modular80

SHA: ~~1c266b8e22ed43ff2c154d5d8e584f2ccdf2531a~~ f1b97e66af4d83ae596ec802f5b8177ee3aaa3db
Tag: 0.6.1

Thank you!

**Edit:** updated SHA to the correct one.

**Comment by mdemanett** [comment]
Hello @cschol!

Everything looks good, but please bump the Makefile version to 0.6.1 so the build scripts will know what to do.

**Comment by cschol** [comment]
Thank you! Fixed in SHA 31008ae3856b13ea0592739312e0a6f5990f5093.

**Comment by AndrewBelt** [comment]
Updated build

**Comment by cschol** [comment]
Updated plugin with `Radio Music` module.

SHA: `27cebc6110a048bfe379694b4d41dd9c5a8a005e`
Version: 0.6.2

Note: the plugin now requires a `git submodule update --init --recursive` before building.

**Comment by AndrewBelt** [comment]
Haha, I've hacked the original Teensy code before because my unit wouldn't turn on when I built mine.

**Comment by cschol** [comment]
It's a reimplementation and Tom liked it enough to make it official. No firmware code is used.

**Comment by AndrewBelt** [comment]
Yeah, I remember it would have not ported well as it relied heavily on the Arduino libraries.

**Comment by cschol** [comment]
@AndrewBelt Can we open this back up so @mdemanett can have a look? Thanks!

**Comment by AndrewBelt** [comment]
Oh, thought I opened it already

**Comment by AndrewBelt** [comment]
Updated build

**Comment by cschol** [comment]
Updated `Radio Music` module. 

- Fixes performance issues with .wav files and excessive logging due to wrongly calculated file position (causing performance issues).
- Implements 2GB bank limit to prevent performance issues when loading files

SHA: e6ea0efad183c61406db951ee411b238818d9606
Version: 0.6.3

**Comment by AndrewBelt** [comment]
Updated build

**Comment by cschol** [comment]
Updated `Radio Music` module.

- Implements setting to enable/disable Sample Looping (like real Radio Music).
- Fixes settings retrieval from json for AllowAllFiles setting

SHA: `29ad56d4189d99060a6f7ef1256f5d43a7779854`
Version: 0.6.4

**Comment by AndrewBelt** [comment]
Updated build

**Comment by cschol** [comment]
Update to v2.0.0: e9a5b103b202deb402238978e48d395af5ed0128

**Comment by cschol** [comment]
Update to v2.0.1: 6020017f6df74d61bb0ccfa7c814257dccf4d853

**Comment by cschol** [comment]
Update to v2.0.2: 2d95796740a64173f6c62ded2fbec52c66667ba7

**Comment by cschol** [comment]
Update to v2.0.3: e786cb894d62b6a72c09a0ac5ed65cc22f4db372

**Comment by cschol** [comment]
Update to v2.0.4: 0db1fe81401c077bab855ebfac368955dc07acaf
